src/backend/services/workflow_service.py

```
@@ -78,7 +78,7 @@
         
         # Run CrewAI workflow (this takes most of the time - 10-15 seconds)
         # User sees progress messages above while this runs
-        validation_output, crew_with_results = run_crew(
+        validation_output, crew_with_results, optimization_metrics = run_crew(
```

Ruff (0.14.5)

Unpacked variable `validation_output` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)